vllm/sampling_params.py

```diff
@@ -28,6 +28,77 @@ class SamplingType(IntEnum):
 to sample from."""
 
 
+class NoBadWordsLogitsProcessor:
```

Comment:
I am not sure if this is the right file for the class. Still I thought it could be placed here near the LogitsProcessor type (the one just above). But if there is a better place for the processor class, I will be ready to move it)